src/main/resources/templates/gcc/activities-class-weekly-schedule.html

```diff
@@ -8,9 +8,9 @@
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <div class="spacing-above-15">
-          <p class="text--grey-dark" th:block th:text="${fieldData.schoolName}"></p>
-          <hr class="spacing-below-25 spacing-above-25">
+        <div class="spacing-above-0">
+          <b><p class="spacing-below-0 text--grey-dark" th:block th:text="${fieldData.schoolName}"></p></b>
+          <hr class="spacing-below-10 spacing-above-0">
```

Comment:
I dont know how to take a screenshot of this but the spacing in Figma is 25
```suggestion
          <hr class="spacing-below-25 spacing-above-0">
```